### PR TITLE
Churches!@ 

### PR DESCRIPTION
**Pull request by JeremyButt**
added church and fixed some bugs

### CODE REVIEW

File: stock-filters/ChirpVillage/Builder.py [hunk 1]
```diff
@@ -41,15 +45,27 @@ def construct_floor_and_flat_roof(level, coords, biome, height):
             utilityFunctions.setBlock(
                 level, floor_block, x, miny, z)
 
+
 def construct_pointed_roof(level, coords, biome, height):
     minx, minz, miny, maxx, maxz = get_coords(coords)
     roof_block = BlockUtils.get_roof_block(biome)
-    for i in range(-1, maxx-minx/4):
+    for i in range(-1, abs(maxx-minx)/4):
```
Comment (johnnyb8) [inline stock-filters/ChirpVillage/Builder.py:52]: this a bug fix?

**Review by johnnyb8 — APPROVED** [review]

**Review by s-axford — APPROVED** [review]
Looks good to me

File: stock-filters/ChirpVillage/Builder.py (hunk 1 above)
Comment (JeremyButt) [reply stock-filters/ChirpVillage/Builder.py:52]: Yeah. In some situations the maxx-minx would be negative.

**Comment by JeremyButt** [comment]
Merging now

**Pull request merged**